app/sqaws.py

```diff
@@ -32,6 +32,8 @@ class Instance:
     monthly_price: float
     monthly_server_price: float
     monthly_storage_price: float
+    stop_after_str: str
```

Comment:
Consider naming these `stop_after_tag_name`; I think that better gets at their true purpose